src/sbt-test/sbt-sql/simple/build.sbt

```diff
@@ -0,0 +1,9 @@
+enablePlugins(SQL)
+
+lazy val root = (project in file(".")).
+  settings(
+    name := "sbt-sql-test",
+    version := "0.2-SNAPSHOT",
+    jdbcUser := "<Your API KEY>"
```

Comment:
`sys.env("TD_API_KEY")` is better for now. If we support more database types (e.g., PostgreSQL) we should change it.